packages/adk/lib/util/space.ts

```diff
@@ -0,0 +1,19 @@
+import fs from 'fs';
+export const ACTION_CONFIG_FILE = '.actionconfig';
+
+export function read_space(): string {
```

Comment:
haven't we been using camel case instead?

Reply:
addressed